### ISSUE TITLE
Fix accessibility violations with React Select (Storybook)

### ISSUE THREAD

**Issue by timdujardin** [ioulian/next-ui-ghent-app-starter#18]
Ensure every form element has an associated label.

https://dequeuniversity.com/rules/axe/4.9/label?application=axeAPI

![image](https://github.com/ioulian/next-ui-ghent-app-starter/assets/5878634/a0d493b4-3526-4b9a-b438-07aaf5d8a786)

**Comment by ioulian** [comment]
problem with react select
